packages/@lwc/engine-dom/src/apis/create-element.ts

```diff
@@ -63,6 +64,9 @@ function monkeyPatchDomAPIs() {
             return callNodeSlot(appendedNode, ConnectingSlot);
         },
         insertBefore(newChild, referenceNode) {
+            if (arguments.length < 2) {
+                logWarnOnce('insertBefore should be called with 2 arguments. Calling with only 1 argument is not supported.');
```

Comment:
```suggestion
                console.warn('insertBefore should be called with 2 arguments. Calling with only 1 argument is not supported.');
```

I just realized that `logWarnOnce` is only available in `@lwc/engine-core`, not `@lwc/engine-dom` (hence `yarn build` is failing). It's fine to just use `console.warn` in this case.